packages/backend/src/open_payments/grant/model.ts

```diff
@@ -25,19 +25,20 @@ export class Grant extends BaseModel {
   }
 
   public authServerId!: string
+  public authServer!: AuthServer
```

Comment:
To be on the safe side, we _could_ keep this as optional, just in case there is no `withGraphFetched` on a `Grant` query, and make `getManagementUrl` an `async` call. 

Similar discussion here: https://github.com/interledger/rafiki/pull/1024#issuecomment-1405432584

Reply:
I vote
```suggestion
  public authServer?: AuthServer

  public getManagementUrl(authServerUrl: string): string {
    return `${authServerUrl}/token/${this.managementId}`
  }
```

afaict, all the places we're calling this have the `paymentPointer.authServer` url
which might let us remove some or all `.withGraphFetched('authServer')